.github/workflows/code_formatting_checks.yml

```diff
@@ -20,28 +20,35 @@ jobs:
         run: ./gradlew spotlessApply
 
       - name: Commit and Push
+        id: commit_and_push
         run: .github/scripts/commit_and_push.sh "${{secrets.USER_NAME}}" "${{secrets.USER_EMAIL}}"
 
       - name: Create PR
         uses: actions/github-script@0.3.0
+        env:
+          branch_name: ${{steps.commit_and_push.outputs.branch_name}}
         with:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
-            var monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
-            var date = new Date();
-            var monthName = monthNames[date.getMonth()];
-            var day = `${date.getDate()}`.padStart(2, 0);
+            const monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
+            const date = new Date();
+            const monthName = monthNames[date.getMonth()];
+            let day = `${date.getDate()}`.padStart(2, '0');
             day = `${monthName}-${day}-${date.getFullYear()}`
-            var branchName = `bot/${day}/code-formatting`
-
-            const branch = await github.repos.getBranch({
-                ...context.repo,
-                branch: branchName
-            });
-            await github.pulls.create({
-                ...context.repo,
-                title: `[BOT] Code formatting fixes ${day}`,
-                head: branch.data.name,
-                base: 'master'
-            });
 
+            const branchName = process.env.branch_name
+
+            if (branchName) {
+                const branch = await github.repos.getBranch({
+                    ...context.repo,
+                    branch: branchName
+                });
+                await github.pulls.create({
+                    ...context.repo,
+                    title: `[BOT] Code formatting fixes ${day}`,
+                    head: branch.data.name,
+                    base: 'master'
+                });
+            } else {
+                console.log("Branch name is empty!")
```

Comment:
Is this necessary? Can we just remove the `else` block?

Reply:
Sure